src/Components/ScoreBar.js

```diff
@@ -4,9 +4,15 @@ class ScoreBar extends Component {
   render() {
     return (
       <div className="score-bar">
-        <span className="score-bar__incorrect-answers"></span>
-        <span className="score-bar__character-number"></span>
-        <span className="score-bar__correct-answers"></span>
+        <span className="score-bar__incorrect-answers">
+          {this.props.counter.incorrectAnswers.length}
```

Comment:
Let's make it shorter and put some variables on top of render method to not make JSX suffer decreased readability